app/components/global-footer-wikia-org.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+	tagName: 'footer',
+	classNames: 'wds-global-footer-wikia-org',
+	year: new Date().getFullYear(),
```

Comment:
It may cause an issue at the turn of the years. I think `new Date().getFullYear()` will be run only once per deployment on fastboot. But I think no one will notice such issue. ;) Just wanted to point it out because in some situations it may be more prominent.